src/app.rs

```diff
@@ -10,6 +10,15 @@ pub struct App<COMP: Component> {
     scope: Scope<COMP>,
 }
 
+impl<COMP> Default for App<COMP>
+where
+    COMP: Component<Properties = ()> + Renderable<COMP>,
```

Comment:
Please remove the `Properties = ()` guard. This was just removed from `new` 😉

Reply:
Fixed